backend/src/core/middlewares/campaign.middleware.ts

```diff
@@ -68,13 +68,16 @@ const listCampaigns = async (
   try {
     const { offset, limit } = req.query
     const userId = req.session?.user?.id
-    const campaigns = await CampaignService.listCampaigns({
+    const { rows, count } = await CampaignService.listCampaigns({
       offset,
       limit,
       userId,
     })
 
-    return res.json(campaigns)
+    return res.json({
```

Comment:
Oh while you're at it, could you also set a maximum on the joi validator for `limit`?

Reply:
Thanks for the catch! Will remember to update swagger the next time round. 

Taking reference from [Stripe's API pagination](https://stripe.com/docs/api/pagination#pagination-limit), would 100 be a reasonable max value to set for `limit`? Along the same vein, do you think we should set a default value for `limit` and `offset` instead of making them optional?

Reply:
Ya, sure